wazo_calld/ari_.py

```diff
@@ -244,13 +244,21 @@ def init_client(self):
 
     def _subscribe_to_bus_events(self):
         for event_name in ALL_STASIS_EVENTS:
+            self._bus_consumer.subscribe(
```

Comment:
Should we check if debug is enabled before subscribing

Reply:
Good idea! I'll do that

Reply:
Actually, since the debug setting can be changed on-the-fly, it would be harder to implement, and the usefulness reduced on production systems. I propose to leave the code as-is, and leave the decision to log to the logging system.